atomic_reactor/plugins/pre_pin_operator_digest.py

```diff
@@ -67,8 +62,7 @@ def __init__(self, workflow, replacement_pullspecs=None,
         Initialize pin_operator_digests plugin
 
         :param workflow: DockerBuildWorkflow instance
-        :param replacement_pullspecs: Dict[str, str], computed in orchestrator,
-                                      provided to workers by osbs-client
+        :param replacement_pullspecs: Dict[str, str]
```

Comment:
This param can be removed, and ideally the `self.replacement_pullspecs` attribute as well (now, `replacement_pullspecs` will be just be computed in the run method)

Reply:
updated.